config/v1/types_infrastructure.go

```diff
@@ -1010,6 +1010,22 @@ type VSpherePlatformTopology struct {
 	// +kubebuilder:validation:Pattern=`^/.*?/vm/.*?`
 	// +optional
 	Folder string `json:"folder,omitempty"`
+
+	// template is the full inventory path of the virtual machine or template
+	// that will be cloned when creating new machines in this failure domain.
+	// The maximum length of the path is 2048 characters.
+	//
+	// if not defined, the template will be calculated by the control plane
```

Comment:
Nit for idiomatic language
```suggestion
	// When omitted, the template will be calculated by the control plane
```